src/test/scala/au/com/agiledigital/toolform/command/generate/kubernetes/VolumeClaimWriterTest.scala

```diff
@@ -0,0 +1,150 @@
+package au.com.agiledigital.toolform.command.generate.kubernetes
+
+import java.io.{StringWriter}
+
+import au.com.agiledigital.toolform.command.generate.kubernetes.VolumeClaimWriter._
+import au.com.agiledigital.toolform.command.generate.WriterContext
+import au.com.agiledigital.toolform.model._
+import org.scalatest.{FlatSpec, Matchers, PrivateMethodTester}
+
+// import scala.io.Source
+// import collection.JavaConverters._
+import com.typesafe.config.ConfigFactory
+
+class VolumeClaimWriterTest extends FlatSpec with Matchers with PrivateMethodTester {
+  "writeAccessModes" should "write access mode if a correct access mode is defined" in {
+    val settings = """"settings": {
+                      |  "accessModes": ["ReadWriteOnce"]
+                      |  "paths": ["/var/lib/mount"]
+                      |}""".stripMargin
+
+    val settingsParam: Option[com.typesafe.config.Config] = Some(ConfigFactory.load(settings))
+    val testResource: Resource = {
+      val testResource = new Resource("ID", "disk", settingsParam, storage = Option("2Gi"))
```

Comment:
Does this nesting serve any purpose, or can we just do the one line
`val testResource = new Resource("ID", "disk", settingsParam, storage = Option("2Gi"))`

Reply:
Good pickup. It served no purpose and has been changed as you suggested.